config.json

```diff
@@ -140,7 +146,7 @@
         {
           "coinType": 0,
           "descriptor": 1,
-          "purpose": 49
+          "purpose": 88
```

Comment:
84

Reply:
Rookie Mistake, thanks for the good catch!